components/hab/src/license.rs

```diff
@@ -160,12 +172,13 @@ fn env_var_present() -> Result<bool> {
 fn write_license_file() -> Result<()> {
     let license = LicenseData::new();
     let content = serde_yaml::to_string(&license)?;
-    fs::create_dir_all(license_path())?;
-    let mut file = File::create(license_file())?;
+    fs::create_dir_all(writeable_license_path())?;
+    let mut file = File::create(license_file(&writeable_license_path()))?;
     file.write_all(content.as_bytes())?;
     Ok(())
 }
 
-fn license_file() -> PathBuf { license_path().join("habitat") }
-
-pub fn license_exists() -> bool { license_file().is_file() }
+pub fn license_exists() -> bool {
+    license_file(&license_path(&superuser_license_root())).is_file()
+    || license_file(&license_path(&user_license_root())).is_file()
```

Comment:
Though I doubt it ends up making a huge difference, it seems like we should check the user license first, then the superuser license, since that's going from more specific to more general.

Reply:
Sure, why not